pennylane/_qubit_device.py

```diff
@@ -280,6 +280,9 @@ def statistics(self, observables):
             elif obs.return_type is Probability:
                 results.append(self.probability(wires=obs.wires))
 
+            elif obs.return_type is State:
+                results.append(self.state)
```

Comment:
don't forget to check if `self.state` is defined 😆 or better yet, try/except

Reply:
Thanks, done!